### PR TITLE
More Eminence Changes

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Eminence now has a radio
Eminence can be given encryption keys by tapping them on the relay pylon
Eminence button act is now fixed
Stargazer's possession enchantment now actually works
Eminence can now see their cog count in the stat pane
Eminence can no longer trigger rad storm
Captain's Encryption Key turns on all channel listen when placed in a headset, not just command and sec

## Why It's Good For The Game

- Eminence now has a radio
All the other clockies get one, the supreme mechanical entity should be able to receive radio waves
- Eminence can be given encryption keys by tapping them on the relay pylon
Interaction, it's not just "here have common" and then done
- Eminence button act is now fixed
Didn't work before
- Stargazer's possession enchantment now actually works
As cool as being a talking pillar is, I think I prefer to be a talking battle hammer
- Eminence can now see their cog count in the stat pane
How am I supposed to judge what a good expenditure of resources is if I can't see the resource count
- Eminence can no longer trigger rad storm
Completely invalidating the prep phase isn't fun for anyone, good riddance

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>
Tested it but didnt screenshot it

Put screenshots and Videos documenting testing and execution of intended behaviors here

</details>

## Changelog
:cl:
add: Eminence now has a radio
add: Eminence can be given encryption keys by tapping them on the relay pylon
fix: Eminence button act is now fixed
fix: Stargazer's possession enchantment now actually works
add: Eminence can now see their cog count in the stat pane
del: Eminence can no longer trigger rad storm, good riddance
/:cl:

### CODE REVIEW

**Comment by unit0016** [comment]
_Eminence can be given encryption keys by tapping them on the relay pylon_

Imagine the absolute **fear** of playing AI and then just hearing the Eminence speak on binary.

**Comment by Ivniinvi** [comment]
> _Eminence can be given encryption keys by tapping them on the relay pylon_
> 
> Imagine the absolute **fear** of playing AI and then just hearing the Eminence speak on binary.

Binary key does not work to my knowledge, this is intentional

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -82,7 +82,7 @@
 /obj/item/encryptionkey/heads/captain
 	name = "\proper the captain's encryption key"
 	icon_state = "cap_cypherkey"
-	channels = list(RADIO_CHANNEL_COMMAND = 1, RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_ENGINEERING = 0, RADIO_CHANNEL_SCIENCE = 0, RADIO_CHANNEL_MEDICAL = 0, RADIO_CHANNEL_SUPPLY = 0, RADIO_CHANNEL_SERVICE = 0, RADIO_CHANNEL_EXPLORATION = 0)
+	channels = list(RADIO_CHANNEL_COMMAND = 1, RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_ENGINEERING = 1, RADIO_CHANNEL_SCIENCE = 1, RADIO_CHANNEL_MEDICAL = 1, RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_EXPLORATION = 1)
```
Comment (AnCopper) [inline code/game/objects/items/devices/radio/encryptionkey.dm]: This is off by default because the Captain is not meant to micromanage the entire station.

Comment (Archanial) [reply code/game/objects/items/devices/radio/encryptionkey.dm]: I agree, revert this change.

File: code/modules/antagonists/clock_cult/mobs/eminence.dm [hunk 2]
```diff
@@ -128,6 +130,14 @@
 	E.runEvent()
 	SSevents.reschedule()
 
+/mob/living/simple_animal/eminence/get_stat_tab_status()
+	var/list/tab_data = ..()
+	tab_data["Cogs Available"] = GENERATE_STAT_TEXT("[cogs] Cogs")
+	return tab_data
```
Comment (Archanial) [inline code/modules/antagonists/clock_cult/mobs/eminence.dm:136]: in perfect world you would add UI element that displays the cogs

**Review by Archanial — APPROVED** [review]

**Pull request merged**